test/sanitizer_suppressions/tsan

```diff
@@ -12,17 +12,22 @@ mutex:CConnman::ThreadOpenAddedConnections
 mutex:CConnman::SocketHandler
 mutex:UpdateTip
 mutex:PeerLogicValidation::UpdatedBlockTip
+mutex:g_best_block_mutex
 # race (TODO fix)
 race:CConnman::WakeMessageHandler
 race:CConnman::ThreadMessageHandler
 race:fHaveGenesis
 race:ProcessNewBlock
 race:ThreadImport
+race:WalletBatch::WriteHDChain
 race:zmq::*
 race:bitcoin-qt
 # deadlock (TODO fix)
 deadlock:CConnman::ForNode
+deadlock:CConnman::GetNodeStats
+deadlock:CChainState::ConnectTip
 deadlock:UpdateTip
+deadlock:wallet_tests::CreateWalletFromFile
```

Comment:
Thanks for this. There's discussion about this case in https://github.com/bitcoin/bitcoin/issues/19049